scripts/before_install.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+cd /opt/unicorn
+kill -9 'cat save_pid.txt'
```

Comment:
if you place the pid file in the more traditional place, just reference the file (no need to `cd` before hand)

Also, good idea to check if the file exists before hand else the script will fail and ever install first time.
Also `$()` is generally preferred over back ticks because it has the ability to nest: 
```
PID_FILE="/var/run/unicorn.pid"
if [[ -z ${PID_FILE} ]]; then
    kill -9 $(cat /var/run/unicorn.pid)
fi
```

*** untested but it _should_ work